### ISSUE TITLE
Project nav isn't marked up as a list of links

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#4380]
## Expected behavior
VoiceOver should announce the project navigation as a list, with the number of items, and give the option to skip past the list to the main content of the page.

## Current behavior
Project nav is marked up as bare links. It should be an unordered list for better navigation by screenreaders.
